[ 2022-04-27T19:56:34     INFO ] logging to applications insights
[ 2022-04-27T19:56:34     INFO ] [ exists                   ] True
[ 2022-04-27T19:56:35     INFO ] [ get                      ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS', 'name': 'VMSSReImageRun1SOUTHCENTRALUS', 'type': 'Microsoft.Resources/resourceGroups', 'properties': <azure.mgmt.resource.resources.v2021_04_01.models._models_py3.ResourceGroupProperties object at 0x7fd31c2d5400>, 'location': 'southcentralus', 'managed_by': None, 'tags': None}
[ 2022-04-27T19:56:37     INFO ] [ create                   ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS/providers/Microsoft.Storage/storageAccounts/vmsspipeline9fc2a6bc', 'name': 'vmsspipeline9fc2a6bc', 'type': 'Microsoft.Storage/storageAccounts', 'tags': {'confidentiality': 'public'}, 'location': 'southcentralus', 'sku': <azure.mgmt.storage.v2021_09_01.models._models_py3.Sku object at 0x7fd31c0f8d90>, 'kind': 'StorageV2', 'identity': None, 'extended_location': None, 'provisioning_state': 'Succeeded', 'primary_endpoints': <azure.mgmt.storage.v2021_09_01.models._models_py3.Endpoints object at 0x7fd31c0f8af0>, 'primary_location': 'southcentralus', 'status_of_primary': 'available', 'last_geo_failover_time': None, 'secondary_location': None, 'status_of_secondary': None, 'creation_time': datetime.datetime(2022, 4, 26, 16, 3, 34, 184382, tzinfo=<isodate.tzinfo.Utc object at 0x7fd31ee0c610>), 'custom_domain': None, 'sas_policy': None, 'key_policy': None, 'key_creation_time': <azure.mgmt.storage.v2021_09_01.models._models_py3.KeyCreationTime object at 0x7fd31c106070>, 'secondary_endpoints': None, 'encryption': <azure.mgmt.storage.v2021_09_01.models._models_py3.Encryption object at 0x7fd31c106340>, 'access_tier': 'Hot', 'azure_files_identity_based_authentication': None, 'enable_https_traffic_only': True, 'network_rule_set': <azure.mgmt.storage.v2021_09_01.models._models_py3.NetworkRuleSet object at 0x7fd31c1065e0>, 'is_sftp_enabled': None, 'is_local_user_enabled': None, 'is_hns_enabled': None, 'geo_replication_stats': None, 'failover_in_progress': None, 'large_file_shares_state': None, 'private_endpoint_connections': [], 'routing_preference': None, 'blob_restore_status': None, 'allow_blob_public_access': True, 'minimum_tls_version': 'TLS1_0', 'allow_shared_key_access': None, 'enable_nfs_v3': None, 'allow_cross_tenant_replication': None, 'default_to_o_auth_authentication': None, 'public_network_access': None, 'immutable_storage_with_versioning': None, 'allowed_copy_scope': None, 'storage_account_sku_conversion_status': None, 'dns_endpoint_type': None}
[ 2022-04-27T19:56:38     INFO ] [ get                      ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS/providers/Microsoft.Network/networkSecurityGroups/vmssPeregrineTest-nsg', 'name': 'vmssPeregrineTest-nsg', 'type': 'Microsoft.Network/networkSecurityGroups', 'location': 'southcentralus', 'tags': {'confidentiality': 'public'}, 'etag': 'W/"032dcc53-bfb9-4cda-95f8-9d8fe45e7237"', 'security_rules': [<azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fd316a2d160>], 'default_security_rules': [<azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fd316a2d070>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fd316a2d190>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fd316a2d1c0>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fd316a2d1f0>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fd316a2d220>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fd316a2d250>], 'network_interfaces': None, 'subnets': [<azure.mgmt.network.v2021_05_01.models._models_py3.Subnet object at 0x7fd316a2d2b0>], 'flow_logs': None, 'resource_guid': '203bc10e-1f08-45fe-8574-0dda31091c3d', 'provisioning_state': 'Succeeded'}
[ 2022-04-27T19:56:39     INFO ] [ get                      ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS/providers/Microsoft.Network/virtualNetworks/vmssPeregrineTest-vnet', 'name': 'vmssPeregrineTest-vnet', 'type': 'Microsoft.Network/virtualNetworks', 'location': 'southcentralus', 'tags': {'confidentiality': 'public'}, 'extended_location': None, 'etag': 'W/"afdc11b6-3c43-4b31-b872-581f5fe7295f"', 'address_space': <azure.mgmt.network.v2021_05_01.models._models_py3.AddressSpace object at 0x7fd316a2d3d0>, 'dhcp_options': None, 'flow_timeout_in_minutes': None, 'subnets': [<azure.mgmt.network.v2021_05_01.models._models_py3.Subnet object at 0x7fd31c0f81c0>], 'virtual_network_peerings': [], 'resource_guid': 'f9c58b12-e959-47a6-90a3-d01f242a9657', 'provisioning_state': 'Succeeded', 'enable_ddos_protection': False, 'enable_vm_protection': None, 'ddos_protection_plan': None, 'bgp_communities': None, 'encryption': None, 'ip_allocations': None}
[ 2022-04-27T19:56:39     INFO ] [ subnets_list             ] <iterator object azure.core.paging.ItemPaged at 0x7fd316a99c70>
[ 2022-04-27T19:56:41     INFO ] [ get                      ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS/providers/Microsoft.Compute/virtualMachineScaleSets/vmssPeregrineTest', 'name': 'vmssPeregrineTest', 'type': 'Microsoft.Compute/virtualMachineScaleSets', 'location': 'southcentralus', 'tags': {'confidentiality': 'public'}, 'sku': <azure.mgmt.compute.v2021_11_01.models._models_py3.Sku object at 0x7fd31678fd00>, 'plan': None, 'identity': None, 'zones': ['1'], 'extended_location': None, 'upgrade_policy': None, 'automatic_repairs_policy': None, 'virtual_machine_profile': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineScaleSetVMProfile object at 0x7fd31679f250>, 'provisioning_state': 'Succeeded', 'overprovision': None, 'do_not_run_extensions_on_overprovisioned_v_ms': None, 'unique_id': '8348e0b6-fe44-4f7e-81b3-a0dbaa7dfc36', 'single_placement_group': False, 'zone_balance': None, 'platform_fault_domain_count': 1, 'proximity_placement_group': None, 'host_group': None, 'additional_capabilities': None, 'scale_in_policy': None, 'orchestration_mode': 'Flexible', 'spot_restore_policy': None, 'time_created': datetime.datetime(2022, 4, 26, 16, 26, 1, 84949, tzinfo=<FixedOffset '+00:00'>)}
[ 2022-04-27T19:56:41     INFO ] [ get_vms                  ] <iterator object azure.core.paging.ItemPaged at 0x7fd316a58970>
[ 2022-04-27T19:56:41     INFO ] [ get_from_vmss_vms        ] <coroutine object get_from_vmss_vms at 0x7fd3167681c0>
[ 2022-04-27T19:56:41    DEBUG ] Using selector: EpollSelector
[ 2022-04-27T19:56:41     INFO ] [ get_vms                  ] <iterator object azure.core.paging.ItemPaged at 0x7fd31679fdf0>
[ 2022-04-27T19:56:42     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance0H1VQ8', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167565b0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316756790>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756640>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167568e0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756910>], 'patch_status': None}
[ 2022-04-27T19:56:43     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance0RB9YO', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3166e1520>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3166e1700>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3166e15b0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3166e1850>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3166e1880>], 'patch_status': None}
[ 2022-04-27T19:56:44     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance3SYUES', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3166e1cd0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3166e1eb0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3166e1d60>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3166f1040>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3166f1070>], 'patch_status': None}
[ 2022-04-27T19:56:45     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance457RRD', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167562e0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167568e0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3167569a0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756430>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756970>], 'patch_status': None}
[ 2022-04-27T19:56:45     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance53CLNL', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756c40>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167510a0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756be0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167511f0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751220>], 'patch_status': None}
[ 2022-04-27T19:56:46     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance5D58EP', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd31674dfd0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd31674dfa0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd31674d3d0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167512e0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751310>], 'patch_status': None}
[ 2022-04-27T19:56:47     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance5K9YCQ', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756f40>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751070>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756700>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167513d0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751400>], 'patch_status': None}
[ 2022-04-27T19:56:47     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance7JHJ5B', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756340>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751190>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756b80>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167514c0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167514f0>], 'patch_status': None}
[ 2022-04-27T19:56:48     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceAA7RWS', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751160>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751100>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751280>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167515b0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167515e0>], 'patch_status': None}
[ 2022-04-27T19:56:48     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceB9R1NT', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167565b0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751310>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756070>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167516a0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167516d0>], 'patch_status': None}
[ 2022-04-27T19:56:49     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceBJGRPR', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751490>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167510d0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751280>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751700>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751730>], 'patch_status': None}
[ 2022-04-27T19:56:50     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceCHWLGH', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd31674da00>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751250>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751190>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751760>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751790>], 'patch_status': None}
[ 2022-04-27T19:56:50     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceCIUBDR', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167510d0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167515b0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751340>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751850>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751880>], 'patch_status': None}
[ 2022-04-27T19:56:51     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceCMAHQN', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751250>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167515e0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751580>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751940>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751970>], 'patch_status': None}
[ 2022-04-27T19:56:52     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceCZQ02Z', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751490>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751430>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3167512b0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751a30>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751a60>], 'patch_status': None}
[ 2022-04-27T19:56:52     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceDJD7CJ', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167513a0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751640>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3167515b0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751b20>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751b50>], 'patch_status': None}
[ 2022-04-27T19:56:53     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceDXNSQX', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751820>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167512e0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751550>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751c10>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751c40>], 'patch_status': None}
[ 2022-04-27T19:56:54     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceEHP95U', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751a30>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751370>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751160>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751d00>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751d30>], 'patch_status': None}
[ 2022-04-27T19:56:55     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceEQYP2Z', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751850>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751b50>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751280>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751e20>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751e50>], 'patch_status': None}
[ 2022-04-27T19:56:55     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceF0MK9Z', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167515e0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751580>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751760>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751f10>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751f40>], 'patch_status': None}
[ 2022-04-27T19:56:56     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceFEOYRA', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167510d0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751a30>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751400>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3166f1040>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3166f1070>], 'patch_status': None}
[ 2022-04-27T19:56:57     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceFM5DBG', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751d00>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167514f0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751070>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3166f1130>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3166f1160>], 'patch_status': None}
[ 2022-04-27T19:56:57     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceFVL3NH', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167563d0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167566a0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3167565b0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3166f1220>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3166f1250>], 'patch_status': None}
[ 2022-04-27T19:56:58     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceFX7RZM', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751520>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167517f0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3167514c0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167566d0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756d60>], 'patch_status': None}
[ 2022-04-27T19:56:59     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceGVTZ0W', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756d30>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316756eb0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756100>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674da90>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674d430>], 'patch_status': None}
[ 2022-04-27T19:56:59     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceH0PFKM', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756b20>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316756970>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756760>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167565e0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756a60>], 'patch_status': None}
[ 2022-04-27T19:57:00     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceIU2LSF', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756460>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316756580>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756070>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674da90>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674d430>], 'patch_status': None}
[ 2022-04-27T19:57:00     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceJ09R5S', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756f70>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316756ac0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756d00>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674d640>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674da00>], 'patch_status': None}
[ 2022-04-27T19:57:01     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceJC00ZJ', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167518b0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167511c0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751bb0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674d1f0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674de20>], 'patch_status': None}
[ 2022-04-27T19:57:02     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceJGZGSD', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756280>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd31674d970>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756bb0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674ddc0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674d2b0>], 'patch_status': None}
[ 2022-04-27T19:57:02     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceJQH4AO', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756df0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd31674de20>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756e50>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674d9d0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674d910>], 'patch_status': None}
[ 2022-04-27T19:57:03     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceL6CGGU', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751700>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd31674da00>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3167511f0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674dca0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674dd60>], 'patch_status': None}
[ 2022-04-27T19:57:04     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceMSR0V1', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751490>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd31674da90>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd31674d3d0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674df10>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674d5e0>], 'patch_status': None}
[ 2022-04-27T19:57:04     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceMUGZIV', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd31674d970>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd31674d2b0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd31674dd60>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674d190>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674d070>], 'patch_status': None}
[ 2022-04-27T19:57:05     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceN8RY7M', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd31674d0a0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd31674de20>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd31674d130>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674d490>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674d4f0>], 'patch_status': None}
[ 2022-04-27T19:57:06     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceNXG7MR', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd31674d2b0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd31674dcd0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd31674ddc0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674d940>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674dac0>], 'patch_status': None}
[ 2022-04-27T19:57:06     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceO1KFR1', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd31674d250>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd31674d850>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd31674d0a0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674d0d0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674ddf0>], 'patch_status': None}
[ 2022-04-27T19:57:07     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstancePXF6CZ', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd31674dfa0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd31674de50>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd31674d070>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674dd90>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674d730>], 'patch_status': None}
[ 2022-04-27T19:57:08     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceQ1R17F', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd31674d280>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd31674d940>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd31674de80>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674d6d0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674db80>], 'patch_status': None}
[ 2022-04-27T19:57:09     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceQBYSO7', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd31674d700>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd31674d640>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd31674dd30>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674df70>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674d580>], 'patch_status': None}
[ 2022-04-27T19:57:09     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceRDL4V3', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd31674d310>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd31674daf0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd31674d490>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757070>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167570a0>], 'patch_status': None}
[ 2022-04-27T19:57:10     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceRJBI1P', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd31674dac0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd31674d940>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd31674d9d0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757160>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757190>], 'patch_status': None}
[ 2022-04-27T19:57:10     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceRMRQB7', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756250>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316756cd0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756910>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757250>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757280>], 'patch_status': None}
[ 2022-04-27T19:57:11     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceSO9I3W', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd31674d070>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757040>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd31674d190>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757340>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757370>], 'patch_status': None}
[ 2022-04-27T19:57:12     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceSPL0HS', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167567c0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757280>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3167561c0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757430>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757460>], 'patch_status': None}
[ 2022-04-27T19:57:12     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceTHBMPT', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd31674d970>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757160>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd31674d400>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757520>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757550>], 'patch_status': None}
[ 2022-04-27T19:57:13     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceUL08X6', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd31674dcd0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757070>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316757310>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757610>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757640>], 'patch_status': None}
[ 2022-04-27T19:57:14     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceVUCYMG', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167572b0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167571f0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3167573d0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757700>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757730>], 'patch_status': None}
[ 2022-04-27T19:57:14     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceX2WAXT', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd31674d6d0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd31674d760>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd31674d4c0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674dd90>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674d940>], 'patch_status': None}
[ 2022-04-27T19:57:15     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceXZQ4EN', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167575b0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757040>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316757490>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757130>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167574f0>], 'patch_status': None}
[ 2022-04-27T19:57:16     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceYCB2V0', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167568e0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167566d0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3167566a0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167576d0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757760>], 'patch_status': None}
[ 2022-04-27T19:57:16     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceYCTNG5', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316757040>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757400>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3167571f0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167573a0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167571c0>], 'patch_status': None}
[ 2022-04-27T19:57:17     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceYV0RIB', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd31674d1f0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167573d0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd31674dc70>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757790>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167577c0>], 'patch_status': None}
[ 2022-04-27T19:57:17     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceZEAPKL', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316757730>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757130>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316757550>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167578b0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757880>], 'patch_status': None}
[ 2022-04-27T19:57:18     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceZ8T8XB', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167573d0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757070>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316757640>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757910>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757940>], 'patch_status': None}
[ 2022-04-27T19:57:19     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceZ18AM3', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316757220>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757610>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316757730>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757a00>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757a30>], 'patch_status': None}
[ 2022-04-27T19:57:19     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceZ0ONQ5', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316757160>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167577f0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316757310>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757af0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757b20>], 'patch_status': None}
[ 2022-04-27T19:57:20     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceYM5XCK', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167572e0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757910>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3167573a0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757be0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757c10>], 'patch_status': None}
[ 2022-04-27T19:57:21     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceXL47N0', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167571c0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757370>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316757400>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757cd0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757d00>], 'patch_status': None}
[ 2022-04-27T19:57:22     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceWRJZIM', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316757b20>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167579d0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316757160>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757dc0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757df0>], 'patch_status': None}
[ 2022-04-27T19:57:22     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceW34T7H', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316757940>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757910>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3167574f0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757eb0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757ee0>], 'patch_status': None}
[ 2022-04-27T19:57:23     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceS6MWC7', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167571c0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757340>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3167573a0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757fa0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757fd0>], 'patch_status': None}
[ 2022-04-27T19:57:23     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstancePWUL7V', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316757670>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167578e0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316757610>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3166f40d0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd31674da30>], 'patch_status': None}
[ 2022-04-27T19:57:24     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstancePOVYID', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd31674de50>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd31674d3a0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd31674dc40>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756ca0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756a30>], 'patch_status': None}
[ 2022-04-27T19:57:25     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceOGFBLB', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd31674d910>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd31674d070>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd31674da90>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167568e0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756fd0>], 'patch_status': None}
[ 2022-04-27T19:57:25     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceOF81WC', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316757ee0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757ac0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316757760>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756250>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756cd0>], 'patch_status': None}
[ 2022-04-27T19:57:26     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceM0OSYW', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167570a0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316756520>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316757d30>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167561c0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756f40>], 'patch_status': None}
[ 2022-04-27T19:57:26     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceKGCQBX', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd31674dfd0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316756cd0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd31674dfa0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756a90>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756130>], 'patch_status': None}
[ 2022-04-27T19:57:27     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceK6J38I', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167571c0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167568e0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3167573d0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756760>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756b20>], 'patch_status': None}
[ 2022-04-27T19:57:28     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceJZNYK9', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316757c40>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316756ca0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756f70>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756c10>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167560d0>], 'patch_status': None}
[ 2022-04-27T19:57:28     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceJUOU0G', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167562e0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316756af0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756550>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756d90>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756670>], 'patch_status': None}
[ 2022-04-27T19:57:29     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceJQBD8L', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756910>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316756cd0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3167567c0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756340>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316756790>], 'patch_status': None}
[ 2022-04-27T19:57:29     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceJ1VZGI', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756580>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167565e0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756eb0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751580>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751880>], 'patch_status': None}
[ 2022-04-27T19:57:30     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceIPOVFQ', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756c40>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167578e0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756ee0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757dc0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757550>], 'patch_status': None}
[ 2022-04-27T19:57:31     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceHT7RVE', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756910>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757d60>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756ca0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757a60>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757af0>], 'patch_status': None}
[ 2022-04-27T19:57:32     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceHK7WBY', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316757850>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757280>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316757550>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167572b0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757190>], 'patch_status': None}
[ 2022-04-27T19:57:33     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceGVV7IV', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756df0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757df0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756970>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757220>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757f10>], 'patch_status': None}
[ 2022-04-27T19:57:34     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceG4AMQB', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316757dc0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757610>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316757be0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167576a0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757310>], 'patch_status': None}
[ 2022-04-27T19:57:35     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceFRTSC3', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316757f10>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757580>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316757520>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757d00>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757c40>], 'patch_status': None}
[ 2022-04-27T19:57:36     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceFQ0NL9', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316757130>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167570d0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316757430>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757a00>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757a30>], 'patch_status': None}
[ 2022-04-27T19:57:36     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceFFX131', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316757f10>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167576a0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316757e80>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757760>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316757700>], 'patch_status': None}
[ 2022-04-27T19:57:37     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceF2FB3A', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316757190>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757520>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3167572b0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167573d0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167574f0>], 'patch_status': None}
[ 2022-04-27T19:57:38     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceEOJJQ9', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316757550>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757400>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316757ac0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751430>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751880>], 'patch_status': None}
[ 2022-04-27T19:57:39     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceEKIN12', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756dc0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316756100>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756220>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751340>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751cd0>], 'patch_status': None}
[ 2022-04-27T19:57:39     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceEIXEMI', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316757e80>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316757580>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316757130>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751040>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751790>], 'patch_status': None}
[ 2022-04-27T19:57:40     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceE9ZLRL', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756460>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751580>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756430>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751c10>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751f40>], 'patch_status': None}
[ 2022-04-27T19:57:41     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceE882I2', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167574f0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751430>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3167573d0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751250>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167518b0>], 'patch_status': None}
[ 2022-04-27T19:57:42     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceE1MSI4', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756be0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751cd0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756df0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751f70>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751370>], 'patch_status': None}
[ 2022-04-27T19:57:43     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceBFBQ9W', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756040>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751730>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751e20>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751d00>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751460>], 'patch_status': None}
[ 2022-04-27T19:57:43     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceB7IZB4', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751580>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751f40>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751370>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751310>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751850>], 'patch_status': None}
[ 2022-04-27T19:57:44     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance8S0OAQ', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751460>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751250>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751730>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751d90>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751130>], 'patch_status': None}
[ 2022-04-27T19:57:45     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance8IIQVP', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd3167512b0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751610>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751520>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167519a0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3166fb040>], 'patch_status': None}
[ 2022-04-27T19:57:46     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance89HDPP', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316756490>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167518e0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316756e80>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751850>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751340>], 'patch_status': None}
[ 2022-04-27T19:57:46     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance5RV6JB', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751400>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751ac0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3167519d0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751cd0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751d30>], 'patch_status': None}
[ 2022-04-27T19:57:47     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance531IQV', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751e50>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167519a0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751be0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751700>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751a90>], 'patch_status': None}
[ 2022-04-27T19:57:48     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance4AHVLO', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751df0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751070>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751940>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751880>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751b50>], 'patch_status': None}
[ 2022-04-27T19:57:48     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance49NDAV', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751a30>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751820>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751670>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751a00>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751250>], 'patch_status': None}
[ 2022-04-27T19:57:49     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance1S2W6W', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751d30>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751370>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd3167519a0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751ca0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751430>], 'patch_status': None}
[ 2022-04-27T19:57:50     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance11W1I0', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751c10>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd3167512b0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751b80>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751e20>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751bb0>], 'patch_status': None}
[ 2022-04-27T19:57:50     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance0ISQ5J', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fd316751a00>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fd316751cd0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fd316751850>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd3167516a0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fd316751af0>], 'patch_status': None}
[ 2022-04-27T19:58:52    ERROR ] Task exception was never retrieved
future: <Task finished name='Task-10' coro=<_get() done, defined at /home/crodriguezde/experiments/reimage/imds.py:14> exception=AttributeError("module 'vm' has no attribute 'name'")>
Traceback (most recent call last):
  File "/home/crodriguezde/experiments/reimage/imds.py", line 17, in _get
    logging.info(f'Starting processing for {instance_vm.name}')
AttributeError: module 'vm' has no attribute 'name'
[ 2022-04-27T19:58:52    ERROR ] Task exception was never retrieved
future: <Task finished name='Task-9' coro=<_get() done, defined at /home/crodriguezde/experiments/reimage/imds.py:14> exception=AttributeError("module 'vm' has no attribute 'name'")>
Traceback (most recent call last):
  File "/home/crodriguezde/experiments/reimage/imds.py", line 17, in _get
    logging.info(f'Starting processing for {instance_vm.name}')
AttributeError: module 'vm' has no attribute 'name'
[ 2022-04-27T19:58:52    ERROR ] Task exception was never retrieved
future: <Task finished name='Task-8' coro=<_get() done, defined at /home/crodriguezde/experiments/reimage/imds.py:14> exception=AttributeError("module 'vm' has no attribute 'name'")>
Traceback (most recent call last):
  File "/home/crodriguezde/experiments/reimage/imds.py", line 17, in _get
    logging.info(f'Starting processing for {instance_vm.name}')
AttributeError: module 'vm' has no attribute 'name'
[ 2022-04-27T19:58:52    ERROR ] Task exception was never retrieved
future: <Task finished name='Task-7' coro=<_get() done, defined at /home/crodriguezde/experiments/reimage/imds.py:14> exception=AttributeError("module 'vm' has no attribute 'name'")>
Traceback (most recent call last):
  File "/home/crodriguezde/experiments/reimage/imds.py", line 17, in _get
    logging.info(f'Starting processing for {instance_vm.name}')
AttributeError: module 'vm' has no attribute 'name'
[ 2022-04-27T19:58:52    ERROR ] Task exception was never retrieved
future: <Task finished name='Task-6' coro=<_get() done, defined at /home/crodriguezde/experiments/reimage/imds.py:14> exception=AttributeError("module 'vm' has no attribute 'name'")>
Traceback (most recent call last):
  File "/home/crodriguezde/experiments/reimage/imds.py", line 17, in _get
    logging.info(f'Starting processing for {instance_vm.name}')
AttributeError: module 'vm' has no attribute 'name'
[ 2022-04-27T19:58:52    ERROR ] Task exception was never retrieved
future: <Task finished name='Task-5' coro=<_get() done, defined at /home/crodriguezde/experiments/reimage/imds.py:14> exception=AttributeError("module 'vm' has no attribute 'name'")>
Traceback (most recent call last):
  File "/home/crodriguezde/experiments/reimage/imds.py", line 17, in _get
    logging.info(f'Starting processing for {instance_vm.name}')
AttributeError: module 'vm' has no attribute 'name'
[ 2022-04-27T19:58:52    ERROR ] Task exception was never retrieved
future: <Task finished name='Task-4' coro=<_get() done, defined at /home/crodriguezde/experiments/reimage/imds.py:14> exception=AttributeError("module 'vm' has no attribute 'name'")>
Traceback (most recent call last):
  File "/home/crodriguezde/experiments/reimage/imds.py", line 17, in _get
    logging.info(f'Starting processing for {instance_vm.name}')
AttributeError: module 'vm' has no attribute 'name'
[ 2022-04-27T19:58:52    ERROR ] Task exception was never retrieved
future: <Task finished name='Task-3' coro=<_get() done, defined at /home/crodriguezde/experiments/reimage/imds.py:14> exception=AttributeError("module 'vm' has no attribute 'name'")>
Traceback (most recent call last):
  File "/home/crodriguezde/experiments/reimage/imds.py", line 17, in _get
    logging.info(f'Starting processing for {instance_vm.name}')
AttributeError: module 'vm' has no attribute 'name'
[ 2022-04-27T19:58:52    ERROR ] Task exception was never retrieved
future: <Task finished name='Task-2' coro=<_get() done, defined at /home/crodriguezde/experiments/reimage/imds.py:14> exception=AttributeError("module 'vm' has no attribute 'name'")>
Traceback (most recent call last):
  File "/home/crodriguezde/experiments/reimage/imds.py", line 17, in _get
    logging.info(f'Starting processing for {instance_vm.name}')
AttributeError: module 'vm' has no attribute 'name'
[ 2022-04-27T19:58:52    ERROR ] Task exception was never retrieved
future: <Task finished name='Task-11' coro=<_get() done, defined at /home/crodriguezde/experiments/reimage/imds.py:14> exception=AttributeError("module 'vm' has no attribute 'name'")>
Traceback (most recent call last):
  File "/home/crodriguezde/experiments/reimage/imds.py", line 17, in _get
    logging.info(f'Starting processing for {instance_vm.name}')
AttributeError: module 'vm' has no attribute 'name'
[ 2022-04-27T19:58:52     INFO ] Exit per user request
